src/browser/api/system.js

```diff
@@ -216,6 +217,14 @@ exports.System = {
         });
 
 
+    },
+    createProxySocket: function(options, callback, errorCallback) {
+        createChromiumSocket(Object.assign({}, options, { callback, errorCallback }));
```

Comment:
Can `callback` and `errorCallback` be separate arguments (instead of object's properties)?